apps/browser/src/content/autofill.js

```diff
@@ -641,6 +641,11 @@
                   0 == confirmResult)) ? true : false;
       }
 
+      // Check if sandboxed
+      function isSandboxed() {
+          return self.origin == null || self.origin === 'null';
+      }
```

Comment:
This comment doesn't really tell me anything

```suggestion
      // Detect if within an iframe, and the iframe is sandboxed
      function isSandboxed() {
          // self.origin is 'null' if inside a frame with sandboxed csp or iframe tag
          return self.origin == null || self.origin === 'null';
      }
```

Do we also want to add comments that we have modified the script or are we past that with the git history?